### PR TITLE
Apply a couple of fixes to the daily snapshot job.

### PR DESCRIPTION
**Pull request by osherdp**
* Using updated credentials to GitHub.
* Changing commit message and slack notification text.

/cc @YuviGold 
/hold
(yes, I'll update Jenkins once pushing this)

### CODE REVIEW

File: Jenkinsfile.daily_repositories_snapshot [hunk 1]
```diff
@@ -21,13 +20,13 @@ pipeline {
...
                 sh '''make snapshot'''
-                withCredentials([usernamePassword(credentialsId: 'oshercc_github_cred', usernameVariable: 'GIT_USERNAME', passwordVariable: 'GIT_PASSWORD')]){
+                withCredentials([usernamePassword(credentialsId: 'osherdp-github-creds', usernameVariable: 'GIT_USERNAME', passwordVariable: 'GIT_PASSWORD')]){
                     sh('''
                         if [ "$(git status --porcelain)" ];
                         then
                             commitDate="date +%d-%m-%Y-%H-%M"
                             git config --local credential.helper "!f() { echo username=\\$GIT_USERNAME; echo password=\\$GIT_PASSWORD; }; f"
-                            git commit -am "Daily commit" -am "`$commitDate`"
+                            git commit -am "Automatic update of repositories revision snapshots" -am "`$commitDate`"
```
Comment (omertuc) [inline Jenkinsfile.daily_repositories_snapshot:29]: What are "revision snapshots"?

Comment (osherdp) [reply Jenkinsfile.daily_repositories_snapshot:29]: rephrasing

**Comment by YuviGold** [comment]
Why there's another Jenkinsfile?

**Comment by osherdp** [comment]
@YuviGold vscode just made it weird
fixed

**Review by YuviGold — APPROVED** [review]

**Pull request merged**